packages/remote-feature-flag-controller/src/remote-feature-flag-controller-types.ts

```diff
@@ -51,3 +51,25 @@ export type ServiceResponse = {
   remoteFeatureFlags: FeatureFlags;
   cacheTimestamp: number | null;
 };
+
+/**
+ * Describes the shape of the state object for the {@link RemoteFeatureFlagController}.
+ */
+export type RemoteFeatureFlagControllerState = {
+  /**
+   * The collection of feature flags and their respective values, which can be objects.
+   */
+  remoteFeatureFlags: FeatureFlags;
+  /**
+   * Local overrides for feature flags that take precedence over remote flags.
+   */
+  localOverrides: FeatureFlags;
+  /**
+   * Raw A/B test flag arrays for flags that were processed from arrays to single values.
```

Comment:
Is this always true? Will feature flags always contain A/B test arrays from now on? There don't seem to be any changes to the `FeatureFlags` array, so it seems like the format continues to be quite flexible and accommodate other use cases.